html/global_attributes.json

```diff
@@ -1140,6 +1144,7 @@
               },
               {
                 "version_added": "68",
+                "version_removed": "79",
```

Comment:
Note, this appears to have been an omission when the preference was disabled for android back in v79